Fix issue with servicesLogs

gfenoy opened the pull request:
Fix typo when accessing the `servicesLogs` array and add the `length` parameter required to detect the multiple values in a maps.

Use the correct location for the logs files by using the standard `tmpUrl` parameter from the `main`section.

gfenoy committed:
Add length key to the service_logs section

gfenoy committed:
Fix typo

gfenoy committed:
Fix typo in accessing the dictionary from the array

gfenoy committed:
Use tmpUrl, process identifier and jobid to create the log urls

bbrauzzi commented:
Thank you for your contribution.
If I may, I'd like to suggest adding the following comment to the **handle_outputs** function to clarify the difference between each output files:

```
    def handle_outputs(self, log, output, usage_report, tool_logs):
        """
        Handle the output files of the execution.

        :param log: The application log file of the execution.
        :param output: The output file of the execution.
        :param usage_report: The performance metrics file for the application during execution.
        :param tool_logs: A list of paths to individual workflow step logs from the execution.

        """
```

And a small comment also above the **servicesLogs** variable just to explain what the loop does
```
# link element to add to the get status response
servicesLogs=[.....
```

You can also remove the pass command which is not needed anymore.

Sorry for being picky. Thank you

gfenoy committed:
Add description to the methods

Following instructions provided in the PR, I have made the corresponding modifications.

gfenoy commented:
Thanks for your feedbacks and no worry for being picky.

I have made the requested modifications.
